source/client-side-encryption/tests/README.rst

```diff
@@ -363,13 +367,9 @@ First, perform the setup.
 
 Using ``client_encrypted`` perform the following operations:
 
-#. Insert ``{ "_id": "no_encryption_under_2mib", "unencrypted": <the string "a" repeated (2097152 - 1000) times> }``. (Note 2097152 is 2^21 bytes, or 2 MiB).
-
-   Expect this to succeed.
+#. Insert ``{ "_id": "over_2mib_under_16mib", "unencrypted": <the string "a" repeated 2097152 times> }``.
 
-#. Insert ``{ "_id": "no_encryption_over_2mib", "unencrypted": <the string "a" repeated 2097152 times> }``.
-
-   Expect this to throw an exception due to exceeding the reduced maximum BSON document size.
```

Comment:
Do we have a test to ensure that a command <=16mib before encryption but > maxBsonObjectSize post encryption will fail?

Reply:
Good call. Added two more tests.